SWIPs/swip-honey_to_money.md

```diff
@@ -7,51 +7,66 @@ Type: Standards track
 Category: Core
 Created: 31-07-2019
 ---
+<!--You can leave these HTML comments in your merged SWIP and delete the visible duplicate text guides, they will not appear and may be helpful to refer to if you edit it again. This is the suggested template for new SWIPs. Note that a SWIP number will be assigned by an editor. When opening a pull request to submit your SWIP, please use an abbreviated title in the filename, `SWIP-draft_title_abbrev.md`. The title should be 44 characters or less.-->
 
 ## Simple summary 
-Swarm needs a mechanism to allow fluctuation of the exchange rate between honey (Swarms internal accounting unit) and the currency used to pay. We propose an on-chain price oracle that returns an `(exchange rate,validUntil)` tuple.  
+<!--"If you can't explain it simply, you don't understand it well enough." Provide a simplified and layman-accessible explanation of the SWIP.-->
+Swarm needs a mechanism to set prices which makes sense relative to other (distributed storage) services in the market. As the absolute price of using Swarm is defined by the cost of honey (Swarm's internal accounting unit), this SWIP proposes a minimal interface to enable upgrading the honey prices network-wide for all nodes. The specifics on what an efficient price should reflect and how the update process looks like is *not* part of this SWIP.
 
 ## Abstract 
-The ability to update prices is required in order to:
-* Facilitate experimentation with the absolute price of honey (e.g. start at a low price and gradually increase);
-* Offset fluctuations of the currency used to pay;
-* Lower the price of Swarm in the long run as the costs of operating a Swarm node decreases.
+<!--A short (~200 word) description of the technical issue being addressed.-->
+It is expected that the price of using Swarm will change over time to reflect the changing economic landscape as well as to facilitate experimentation. Such price updates must happen atomically or not at all, to prevent accounting disbalances to appear. To lay the groundwork for enabling such price-changes, this SWIP propose a minimal interface for a *price-oracle*. Furthermore, this SWIP includes a proposal on implementation of an *on-chain* price oracle, governed by Swarm developers and stakeholders to allow for experimentation. 
```

Comment:
typo: "this SWIP proposeS"

"Furthermore, this SWIP includes a proposal for the implementation of..."